.github/workflows/linting.yml

```diff
@@ -155,7 +155,7 @@ jobs:
       - name: Install extra python packages
         run: python3 -m pip install --requirement utils/cq/requirements.txt
       - name: Run check
-        uses: codespell-project/actions-codespell@3174815d6231f5bdc24dbfb6fc3b8caec73d521c  # master
+        uses: codespell-project/actions-codespell@406322ec52dd7b488e48c1c4b82e2a8b3a1bf630  # master
```

Comment:
Do we really want to be following `master` for this action rather than a release stream?

Reply:
You're right. And luckily this current master SHA is a release: https://github.com/codespell-project/actions-codespell/releases/tag/v2.1
So I can just update the comment here to say v2.1 instead, and keep it updated by version